R/htmlwidgets.R

```diff
@@ -329,16 +333,18 @@ createWidget <- function(name,
 #' @name htmlwidgets-shiny
 #'
 #' @export
-shinyWidgetOutput <- function(outputId, name, width, height, package = name) {
+shinyWidgetOutput <- function(outputId, name, width, height, inline = FALSE,
+                              package = name) {
```

Comment:
I think most probably go with the template from `scaffoldWidget` which currently looks and is proposed to look like [this](https://github.com/ramnathv/htmlwidgets/blob/move-templates-to-files/inst/templates/widget_r.txt#L42):

```
#' @export
%sOutput <- function(outputId, width = '100%%', height = '400px'){
  shinyWidgetOutput(outputId, '%s', width, height, package = '%s')
}
```